app/_hub/kong-inc/kafka-log/index.md

```diff
@@ -124,82 +125,12 @@ params:
         API version of a Produce request. Allowed values: `0`, `1`, or `2`.
 ---
 
-## Installation
-
-Manually download and install:
-
-```
-$ git clone https://github.com/kong/kong-plugin-kafka-log.git /path/to/kong/plugins/kong-plugin-kafka-log
-$ cd /path/to/kong/plugins/kong-plugin-kafka-log
-$ luarocks make *.rockspec
-```
-
-In both cases, you need to change your Kong `plugins`
-[configuration option](https://docs.konghq.com/latest/configuration/#plugins)
-to include this plugin:
-
-```
-plugins = bundled,kong-plugin-kafka-log
-```
-
-Or, if you don't want to activate any of the bundled plugins:
-
-```
-plugins = kong-plugin-kafka-log
-```
-
-Then reload kong:
-
-```
-kong reload
-```
-
-## Configuration
-
-### Enabling globally
-
-```bash
-$ curl -X POST http://kong:8001/plugins \
```

Comment:
this is what the autogen example should look like, not have all cap entries in the value in examples parameters fields.